compiler/parser/src/string_parser.rs

```diff
@@ -219,35 +219,19 @@ impl<'a> StringParser<'a> {
                         Some('a') => ConversionFlag::Ascii,
                         Some('r') => ConversionFlag::Repr,
                         Some(_) => {
-                            return Err(if expression.trim().is_empty() {
```

Comment:
We have the following code at line 213:

```rust
                    if expression.trim().is_empty() {
                        return Err(EmptyExpression.to_lexical_error(self.get_pos()));
                    }
```

After this if block, `expression.trim().is_empty()` is always false.